frameworks/cockroachdb/src/main/dist/svc.yml

```diff
@@ -0,0 +1,196 @@
+name: {{FRAMEWORK_NAME}}
+scheduler:
+  principal: {{FRAMEWORK_PRINCIPAL}}
+pods:
+  cockroachdb:
+    count: {{NODE_COUNT}}
+    uris:
+      - {{COCKROACH_URI}}
+      - {{BOOTSTRAP_URI}}
+    {{#ENABLE_VIRTUAL_NETWORK}}
+    networks:
+      dcos:
+    {{/ENABLE_VIRTUAL_NETWORK}}
+    placement: {{NODE_PLACEMENT}}
+    resource-sets:
+      server-resources:
+        cpus: {{NODE_CPUS}}
+        memory: {{NODE_MEM}}
+        ports:
+          gui:
+            port: {{CONTAINER_GUI_PORT}}
+            vip:
+              prefix: gui
+              port: {{COCKROACH_GUI_PORT}}
+          internal:
+            port: {{CONTAINER_INTERNAL_PORT}}
+            vip:
+              prefix: internal
+              port: {{COCKROACH_INTERNAL_PORT}}
+        volume:
+          path: "cockroach-data"
+          type: {{NODE_DISK_TYPE}}
+          size: {{NODE_DISK}}
+      sidecar-resources:
+        cpus: {{SIDE_CPUS}}
+        memory: {{SIDE_MEM}}
+    tasks:
+      node-init:
```

Comment:
This should never run more than once. If you look at the plans at the bottom of this file, the `node-init` task should only be run for Pod 0 during the deploy plan. All restores, upgrades, etc. use `node-join` exclusively to avoid re-initializing the cluster.

Fwiw, even if `node-init` were run again, this should only re-initialize the cluster in the case of a permanent failure where the persistent volume containing the cockroach-data directory is lost. Otherwise Cockroach will recover a list of the nodes to which it was connected and rejoin the cluster. But the permanent failure case is covered and would use `node-join` anyways.